JASP-Engine/JASP/R/reliabilityAnalysisBayesian.R

```diff
@@ -238,26 +252,54 @@ reliabilityBayesian <- function(jaspResults, dataset, options) {
 
 
 .BayesianReliabilityCheckLoadings <- function(dataset, variables) {
-
-	prin <- psych::principal(dataset)
-	idx <- prin[["loadings"]] < 0
-	sidx <- sum(idx)
-	if (sidx == 0) {
-	  return()
-	} else {
-	  hasSchar <- if (sidx == 1L) "" else "s"
-	  footnote <- gettextf("The following item%s correlated negatively with the scale: %s. ",
-	                       hasSchar, paste0(variables[idx], collapse = ", "))
-	  return(footnote)
-	}
-
+  if (ncol(dataset > 2)) {
+    prin <- psych::principal(dataset)
+    idx <- prin[["loadings"]] < 0
+    sidx <- sum(idx)
+    if (sidx == 0) {
+      return("")
+    } else {
+      hasSchar <- if (sidx == 1L) "" else "s"
+      footnote <- gettextf("The following item%s correlated negatively with the scale: %s. ",
+                           hasSchar, paste0(variables[idx], collapse = ", "))
+      return(footnote)
+    }
+  } else {
+    return("")  
+    }
 }
 
 
 # ----------------------------- tables ------------------------------------
 .BayesianReliabilityScaleTable <- function(jaspResults, model, options) {
   
-  if (!is.null(jaspResults[["scaleTable"]])) {
+  if ((!options[["mcDonaldScale"]] & !options[["alphaScale"]] & !options[["guttman2Scale"]] 
+       & !options[["guttman6Scale"]] & !options[["glbScale"]] & !options[["averageInterItemCor"]]
+       & !options[["meanScale"]] & !options[["sdScale"]])) {
+    scaleTable <- createJaspTable(gettext("Bayesian Scale Reliability Statistics"))
+    scaleTable$dependOn(options = c("variables", "reverseScaledItems", "credibleIntervalValueScale"))
+    
+    scaleTable$addColumnInfo(name = "estimate", title = "Estimate", type = "string")
+    intervalLow <- gettextf("%s%% CI",
+                            format(100*options[["credibleIntervalValueScale"]], digits = 3, drop0trailing = TRUE))
+    intervalUp <- gettextf("%s%% CI",
+                           format(100*options[["credibleIntervalValueScale"]], digits = 3, drop0trailing = TRUE))
+    intervalLow <- gettextf("%s lower bound", intervalLow)
+    intervalUp <- gettextf("%s upper bound", intervalUp)
```

Comment:
If you lift this stuff above the `if` statement then you don't have to do the same thing again [under the `if` statement](https://github.com/jasp-stats/jasp-desktop/blob/859eefde7141571d7ed0cf6704df46b7dd35fa4e/JASP-Engine/JASP/R/reliabilityAnalysisBayesian.R#L306-L319).